tools/pylint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# This script can be used to run the pylint_runner while ignoring migrations.
+
+# Import utility functions
+# shellcheck source=./tools/utils/_functions.sh
+source "$(dirname "${BASH_SOURCE[0]}")/utils/_functions.sh"
+
+require_installed
+
+# Run pylint
+echo "Starting code linting with pylint..." | print_info
+pylint ./integreat_compass/*.py
```

Comment:
```suggestion
pylint .
```
I think using the `recursive` option makes more sense than path globbing

Reply:
Yes, I just realized that myself :)